ndex2/client.py

```diff
@@ -720,9 +959,38 @@ def get_task_by_id(self, task_id):
         return self.get(route)
 
     def delete_network(self, network_id, retry=5):
-        """
+        """            
+
         Deletes the specified network from the server
 
+        .. code-block:: python
+            	
+		import ndex2.client
+		my_account="your account"
+		my_password="your password"
+		my_ndex=ndex2.client.Ndex2("http://public.ndexbio.org", my_account, my_password)
+
+
+		# Query the network from my NDEx account, get a small subset
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','ENTER NODE NAME HERE')
```

Comment:
Instead of making the user find a network they can run a neighborhood query on just directly create a network with this fragment of code (you'll have to move the import to the top.

```
from ndex2.nice_cx_network import NiceCXNetwork

my_net = NiceCXNetwork()
my_net.create_node('testnode1')
my_net.set_name('testnetwork')
uri_subset = my_ndex.save_new_network(my_net.to_cx())
```